src/Storage/Session.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace PhpAb\Storage;
+
+use InvalidArgumentException;
+
+/**
+ * Stores the participation state of the user in a session.
+ */
+class Session implements StorageInterface
+{
+    /**
+     * @var string The namespace of the session.
+     */
+    private $namespace;
+
+    /**
+     * Initializes a new instance of this class.
+     *
+     * @param string $namespace The namespace of the session.
+     */
+    public function __construct($namespace)
+    {
+        if (!$namespace) {
+            throw new InvalidArgumentException('The namespace is invalid.');
+        }
+
+        $this->namespace = $namespace;
+    }
+
+    /**
+     * Gets the namespace that is used for this stroage.
+     */
+    public function getNamespace()
```

Comment:
should we make this private or at least protected. It's not needed and not part of the interface.
(if we decide different later it's easy to make it public. Removing from public API is hard)